apps/publications/src/app/components/update-rank-dialog/update-rank-dialog.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="user-theme">
+  <h1 mat-dialog-title>{{"DIALOGS.UPDATE_RANK.TITLE" | translate}}</h1>
```

Comment:
in the title of this dialog I would like to have the name of the category
like: "Edit rank of category Jine"